### PR TITLE
added close-figs context

### PR DESCRIPTION
**Pull request by hyosubkim**
**What is this PR about?**
Added ":context: close-figs" after all instances of ".. plot::"

**Checklist**
+ [ ] Explain important implementation details 👆
+ [ x] Make sure that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html).
+ [ ] Link relevant issues (preferably in [nice commit messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html))
+ [ ] Are the changes covered by tests and docstrings?
+ [ ] Fill out the short summary sections 👇

## Major / Breaking Changes
- ...

## Bugfixes / New features
- ...

## Docs / Maintenance
- ...

### CODE REVIEW

**Comment by hyosubkim** [comment]
@michaelosthege - tried to push the same commits on a different branch since I saw many of the checks had failed. Sorry I don't know why so many tests are failing

File: pymc/distributions/discrete.py [hunk 1]
```diff
@@ -1140,17 +1140,18 @@ def rng_fn_scipy(cls, rng, lower, upper, size=None):
         return stats.randint.rvs(lower, upper + 1, size=size, random_state=rng)
 
 
-discrete_uniform = DiscreteUniformRV()
+discrete_uniform = RV()
```
Comment (michaelosthege) [inline pymc/distributions/discrete.py]:
```suggestion
discrete_uniform = DiscreteUniformRV()
```

Undoing this change should fix the CI pipeline

**Review by michaelosthege — CHANGES_REQUESTED** [review]
glad you got the git problems sorted out!

Thanks for adding the remaining `close-figs`. I commented one thing that should fix the CI pipeline and then we're good to go

File: pymc/distributions/discrete.py (hunk 1 above)
Comment (hyosubkim) [reply pymc/distributions/discrete.py]: Thanks. That seems to be it. Not sure how that change got in there, but I'll check the diffs more carefully next time.

**Review by michaelosthege — APPROVED** [review]
Thanks for the fixes!

merging now...

**Comment by hyosubkim** [comment]
Great. Thanks for the help.

**Pull request merged**